### DISCUSSION TITLE
New Commands

### DISCUSSION THREAD

**Discussion by DvRkW0lF** [Shadow-Creation-Team/Protector#46] [Feature Request & Ideas]
- **Commands χρόνου**
reminder, χρονόμετρο, αντίστροφη μέτρηση κτ.λ

- **Create text channel, category, voice channel**
Όλα τα παραπάνω να είναι σε ένα command και ο author του command να πρέπει να διαλέξει το τι θέλει να φτιάξει με Select Menu κ μετά να πρέπει να δώσει το όνομα που θέλει, που να βρίσκεται, και πάει λέγοντας...

- **Emoji Info Command**

- **Voice Commands**
Voice Kick-Mute-Move-Ban

- **Channel Nuke - Server Nuke** Command
Χρήσιμη εντολή, π.χ Channel Nuke να σβήνει το channel και να φτιάχνει ένα ολόηδιο channel στο ίδιο Position

- **Achievements?**

Suggestion για το *warn*
**Να δίνει ένα role με τον αριθμό των warnings στον warned user**

**Comment by DvRkW0lF** [comment]
Και κάτι που ξέχασα να πω είναι το ότι μπορείτε να βάλετε και *doubleprefix* ( η περισσοτερα ) αλλα και *noprefix* δλδ να απαντάει το bot σε commands χωρίς να χουν μπροστά το **prefix**

**Reply by andreaskapsalhs** [reply]
Αφού το bot θα έχει slashcommands..

**Reply by tougrel** [reply]
Δεν θα υπάρχει prefix από την 0.5 λόγο των slash commands.

**Reply by DvRkW0lF** [reply]
swsta to jexasa :D

**Comment by andreaskapsalhs** [comment]
1) Καλή ιδέα θα τα βάλουμε κάποια στιγμή. (λογικά v0.5.1~)

2) Δεν είναι χρήσιμα κατά την γνώμη μου και δεν νομίζω να μπούνε.

3) Υπάρχει ήδη κάτι παρόμοιο στα πλάνα μας (See here #44)

4) Δεν είναι χρήσιμα υπάρχουν κι'αυτά σαν επιλογή στο discord και είναι πιο accessible.

5) no

6) Θα υπάρχουν μαζί και quests 😄

7) Χμμ, θα βάλουμε κάτι παρόμοιο στο dashboard αλλά όχι τώρα.. (0.5.4~)
